project/iterative_sorting.py

```diff
@@ -6,11 +6,15 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
+        for x in range (cur_index + 1, len(arr)):
+            if arr(x) < arr(smallest_index):
```

Comment:
This function will work if you change this line to `if arr[x] < arr[smallest_index]:` with the square braces instead of parentheses.